sssd_test_framework/misc/__init__.py

```diff
@@ -177,6 +194,38 @@ def seconds_to_timespan(seconds: int) -> str:
     return f"{d:02d}:{h:02d}:{m:02d}:{s:02d}:00"
 
 
+def jc_parse_dig(data: str) -> tuple[dict | None, Any | None]:
```

Comment:
Is the verbose `dig` output needed in most cases, or would `dig +short` output be enough?

Reply:
I reworked the dig command, but shouldn't wouldnt work. the dig exit code is answer part of the answer. Fun fact, most dig commands has an rc of 0.